sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/block/flow/FlowRuleChecker.java

```diff
@@ -194,7 +194,7 @@ private static boolean applyTokenResult(/*@NonNull*/ TokenResult result, FlowRul
                 try {
                     Thread.sleep(result.getWaitInMs());
                 } catch (InterruptedException e) {
-                    e.printStackTrace();
+                    RecordLog.warn("[FlowRuleChecker] Error when applyTokenResult", e);
```

Comment:
InterruptedException is not an error, but indicates that the wait operation has been interrupted.

Reply:
I will change the description.